force-app/main/default/classes/RelationshipSettingsVMapper.cls

```diff
@@ -76,7 +76,10 @@ public virtual with sharing class RelationshipSettingsVMapper {
             relationshipTypeComboboxValueVModelsList
         );
 
-        return new RelationshipSettingsVModel(defaultRelationshipComboboxVModel);
+        return new RelationshipSettingsVModel(
+            defaultRelationshipComboboxVModel,
+            !currentRelationshipSettingsModel.allowAutoCreatedDuplicates
```

Comment:
@MatthewBlanski would this be the right place to handle the inversion of the custom settings value?